src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
-
+enum Figures {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
 }
 
-export class Triangle {
+type Colors = 'red' | 'green' | 'blue';
```

Comment:
Don't name type in plural
```suggestion
type Color = 'red' | 'green' | 'blue';
```